rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -972,17 +972,17 @@ public Integer rollbackFederation(Object[] args) throws BridgeIllegalArgumentExc
         );
     }
 
-    public byte[] getPendingFederationHash(Object[] args) {
+    public byte[] getPendingFederationHashBytes(Object[] args) {
```

Comment:
```suggestion
    public byte[] getPendingFederationHashSerialized(Object[] args) {
```

Reply:
Good one, we shouldn't use data types or structures as part of the names

Reply:
on it! thanks guys

Reply:
done